src/runtime/RegExpObject.cpp

```diff
@@ -532,4 +532,29 @@ void RegExpObject::pushBackToRegExpMatchedArray(ExecutionState& state, ArrayObje
         }
     }
 }
+
+RegExpStringIteratorObject::RegExpStringIteratorObject(ExecutionState& state, bool global, bool unicode, RegExpObject* regexp, String* string)
+    : IteratorObject(state, state.context()->globalObject()->regexpStringIteratorPrototype())
+    , m_isGlobal(global)
+    , m_isUnicode(unicode)
+    , m_isDone(false)
+    , m_regexp(regexp)
+    , m_string(string)
+{
+}
+
+void* RegExpStringIteratorObject::operator new(size_t size)
+{
+    static bool typeInited = false;
+    static GC_descr descr;
+    if (!typeInited) {
+        GC_word obj_bitmap[GC_BITMAP_SIZE(RegExpStringIteratorObject)] = { 0 };
+        Object::fillGCDescriptor(obj_bitmap);
+        GC_set_bit(obj_bitmap, GC_WORD_OFFSET(RegExpStringIteratorObject, m_regexp));
+        GC_set_bit(obj_bitmap, GC_WORD_OFFSET(RegExpStringIteratorObject, m_string));
```

Comment:
Request just one more thing here,
Would you please add `RegExpStringIteratorObject::fillGCDescriptor` function and invoke `Object::fillGCDescriptor` and mark members inside that function?
It's for code consistency.
You can find similar cases, for example `GeneratorObject`'s new operator overloading.